### ISSUE TITLE
add back in dictionary entry 

### ISSUE THREAD

**Issue by gbinal** [GSA/site-scanning#1142]
> Accessibility - Violations,accessibility_violations,Count of Axe rules which have 1+ voilations detected - by category of rule,string,no
